example-usage.ts

```
@@ -1,6 +1,6 @@
 // Imports
 import express, { type Request, type Response } from 'express';
-import { AuthProvider, Environment, ApiClient, AuthScopes, EstimateStatus } from './src/app';
+import { AuthProvider, Environment, ApiClient, AuthScopes, EstimateStatus, Invoice } from './src/app';
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)